Assets/Shopify/Examples/Scripts/Panels/ProductsPanel.cs

```diff
@@ -14,19 +16,30 @@ public class ProductsPanel : MonoBehaviour {
         public ShowProductEvent OnShowProduct = new ShowProductEvent();
         public UnityEvent OnViewCart;
         public UnityEvent OnClosePanel;
+        public UnityEvent OnNetworkError;
 
         public Button ViewCartButton;
         public Button ClosePanelButton;
 
         public ProductsPanelCell ProductsPanelCellTemplate;
 
         private readonly List<ProductsPanelCell> _lineItems = new List<ProductsPanelCell>();
+        private List<string> _addedProductIds = new List<string> ();
 
         public ScrollRect ScrollView;
         public RectTransform Content;
 
         private RectTransform RectTransform;
 
+        private bool HitEndCursor;
+
+        private string After;
+
+        private bool WasScrolledToBottom;
+        private bool IsScrolledToBottom;
+
+        private int ScrollCount;
```

Comment:
we've been prefixing private member vars with a `_`

Reply:
updated!